contest/views.py

```diff
@@ -28,7 +28,7 @@
 from django.template import RequestContext
 from django.forms.models import model_to_dict
 
-from index.views import custom_proc
+from utils.render_helper import custom_proc
```

Comment:
This function is only used inside ``render_index``.
I think it's not necessary to import.